spec/hydra/pcdm/models/collection_spec.rb

```diff
@@ -26,9 +26,13 @@
         subject.members << object1
         subject.members << object2
         subject.members << collection3
+        subject.save
```

Comment:
There should probably be an isolated test for #parent_collection_ids. Something like

```ruby
describe "#parent_collection_ids" do
  it "should return the parent's IDs" do
    subject.members = [object1]
    subject.save!

    expect(object1.parent_collection_ids).to eq [subject.id]
  end
end
```